edfi-repo-auditor/edfi_repo_auditor/checklist.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from collections import namedtuple
+
+options = namedtuple("checklist",
```

Comment:
I have never seen a named tuple used this way 😄 . Seems to be working quite well though, so I have no concern. Typically I have just had plain strings without the tuple. Those strings are not readonly constants though, and I think you have achieved real readonly costants.